src/boardlib/db/aurora.py

```diff
@@ -33,12 +33,25 @@ def download_database(board, output_file):
         headers={
             "User-Agent": "Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/119.0.0.0 Safari/537.36"
         },
+        verify=False,
```

Comment:
What is the purpose of this change?

Reply:
Yeah that shouldnt be merged, I still was struggling to get it working. Then I switched to a curl'ing the apk, and that was a apk in the shape that I expected oddly enough